src/components/OverlayTooltip/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { OverlayTrigger, Tooltip } from 'patternfly-react'
+
+class OverlayTooltip extends React.Component {
+  // constructor (props) {
+  //   super(props)
+  //   var id = props.id
+  //   var tooltip = props.tooltip
+  //   var placement = props.placement
+  //   var children = props.children
+  // }
+  render () {
+    return (
+      <OverlayTrigger
+        overlay={
+          <Tooltip id={this.props.id}>{this.props.tooltip}</Tooltip>
```

Comment:
nit-pick: I think it will be more readable if you use `const { id, tooltip, placement, children } = this.prop`, instead of writing `this.props` every time.